src/pages/kyunghyun/Login/Login.js

```diff
@@ -1,8 +1,174 @@
+    const validId = id.includes('@');
+    const validPw = pw.length >= 6;
+    validId && validPw
+      ? this.setState({ valid: true })
+      : this.setState({ valid: false });
+  };
+  // id input의 @와 pw input이 길이가 6 이상의 문자열을 받는 유효성 확인
+  // valid가 true일 시 className이 변경되어 버튼 색 변경
+
+  loginValidation = () => {
+    const { valid, pw, id, alert } = this.state;
+    const { history } = this.props;
+
+    valid ? history.push('/Main-KyungHyun') : (id = '');
+    pw = '';
+    alert =
+      '입력한 사용자 이름을 사용하는 계정을 찾을 수 없습니다. 사용자 이름을 확인하고 다시 시도하세요.';
+  };
+  //// loginValidation은 유효성 검사 함수 -> 추후 스케줄에 적용 예정
+
+  // enterValidation = e => {
+  //   const { history } = this.props;
+  //   if (e.key === 'Enter' && this.state.valid) {
+  //     history.push('/Main-KyungHyun');
+  //   }
+  // };
+
+  handleSubmit = e => {
+    // const { history } = this.props;
+    const { id, pw, alert, valid } = this.state;
+    const { history } = this.props;
+    // e.preventDefault();
+    if (valid) {
+      // 10.58.4.21:8000/users/signup
+      // http://10.58.5.12:8000/users/login
+      fetch('http://10.58.5.12:8000/users/login', {
+        method: 'POST',
+        body: JSON.stringify({
+          email: this.state.id,
+          password: this.state.pw,
+          // name: 'dfsfdsdf',
+          // phone_number: '01041560647',
+          // information: 'EKFKEKEKEK',
+        }),
+      })
+        .then(res => res.json())
+        .then(res => {
+          if (res.token) {
+            history.push('/Main-KyungHyun');
+            localStorage.setItem('kich-token', res.token);
+          } else {
+            this.setState({
+              id: '',
+              pw: '',
+              alert:
+                '입력한 사용자 이름을 사용하는 계정을 찾을 수 없습니다. 사용자 이름을 확인하고 다시 시도하세요.',
+            });
+          }
+        });
+    }
+  };
+  //submit이 일어날 시  history.push('/Main-KyungHyun'); 실행
+
   render() {
-    return null;
+    const { id, pw, valid } = this.state;
+    return (
+      <div className="KyungHyunLogin">
+        <header>
+          <h1>
+            <b>instagram</b>
+          </h1>
+          <div className="main-function">
+            <form className="login-form" onSubmit={this.handleSubmit}>
+              <input
+                className="id"
+                placeholder="전화번호, 사용자 이름 또는 이메일"
+                type="text"
+                onChange={this.handleInput}
+                onKeyUp={this.checkValid}
+                // onKeyDown={this.enterValidation}
+                value={id}
+                name="id"
+              />
+              <input
+                className="pw"
+                placeholder="비밀번호"
+                type="password"
+                onChange={this.handleInput}
+                onKeyUp={this.checkValid}
+                // onKeyDown={this.enterValidation}
+                value={pw}
+                name="pw"
+              />
+              <button
+                className={valid ? 'valid' : 'login'}
```

Comment:
valid, login이 서로 반대되는 쌍으로 보기에는 어색한 네이밍 같아요 혹시 이런 의도셨나요?
```suggestion
                className={isLoginValid ? 'button active' : 'button'}
```